client/webserver/site/src/js/markets.ts

```diff
@@ -598,24 +596,32 @@ export default class MarketsPage extends BasePage {
 
   /* setHighLow calculates the high and low rates over the last 24 hours. */
   setHighLow () {
-    const cache = this.market?.candleCaches[fiveMinBinKey]
-    if (!cache) {
-      for (const s of this.stats) {
-        s.tmpl.high.textContent = '-'
-        s.tmpl.low.textContent = '-'
+    let [high, low] = [0, 0]
+    const spot = this.market.cfg.spot
+    // Use spot values for 24 hours high and low rates if it is available. We
+    // will default to setting it from candles if it's not.
+    const lowRateSet = spot && spot.low24 !== undefined && spot.low24 !== 0
+    const highRateSet = spot && spot.high24 !== undefined && spot.high24 !== 0
+    if (lowRateSet && highRateSet) {
+      high = spot.high24
+      low = spot.low24
+    } else {
+      const cache = this.market?.candleCaches[fiveMinBinKey]
+      if (!cache) {
+        for (const s of this.stats) {
+          s.tmpl.high.textContent = '-'
+          s.tmpl.low.textContent = '-'
```

Comment:
This will always be the case for old servers on first load, since we're defaulting to one-hour bins now. Maybe something like below ?

```Typescript
if (!cache) {
  if (this.candleDur !== fiveMinBinKey) {
    this.candleDur = fiveMinBinKey
    this.requestCandles()
    return
  }
  for (const s of this.stats) {
    s.tmpl.high.textContent = '-'
    s.tmpl.low.textContent = '-'
  }
  return
}
```
Effectively forcing old servers to default to five-minute bins and ignoring the last user's choice.